h2/src/main/org/h2/mvstore/MVStore.java

```diff
@@ -1853,10 +1950,10 @@ public int getCurrentFillRate() {
             @Override
             public int compare(Chunk o1, Chunk o2) {
                 int comp = Integer.compare(o1.collectPriority,
-                        o2.collectPriority);
+                                            o2.collectPriority);
```

Comment:
Old indentation was better.

Reply:
"Better" is a very subjective term in this context, don't you think? No identation - no problem. 8-)